roles/matrix-dimension/tasks/setup_dimension.yml

```diff
@@ -0,0 +1,79 @@
+---
+
+#
+# Tasks related to setting up the dimension
+#
+
+- name: Ensure dimension base path exists
+  file:
+    path: "{{ matrix_dimension_base_path }}"
+    state: directory
+    mode: 0750
+    owner: "{{ matrix_user_username }}"
+    group: "{{ matrix_user_username }}"
+  when: matrix_dimension_enabled
+
+- name: Ensure dimension environment variables file created
+  template:
+    src: "{{ role_path }}/templates/config.yaml.j2"
+    dest: "{{ matrix_dimension_base_path }}/config.yaml"
+    mode: 0644
```

Comment:
We can do the following to make sure the file is readable by the Dimension container:

```
    mode: 0640
    owner: "{{ matrix_user_username }}"
    group: "{{ matrix_dimension_user_gid }}"
```